packages/testkit-backend/src/skipped-tests/common.js

```diff
@@ -1,8 +1,11 @@
-import skip, { ifEquals, ifEndsWith } from './skip'
+import skip, { ifEquals, ifEndsWith, endsWith, ifStartsWith, startsWith, not } from './skip'
 
 const skippedTests = [
   skip(
     'Driver does not return offset for old DateTime implementations',
+    ifStartsWith('stub.types.test_temporal_types.TestTemporalTypes')
+      .and(not(startsWith('stub.types.test_temporal_types.TestTemporalTypesV5')))
+      .and(endsWith('test_zoned_date_time')),
```

Comment:
Is there no way to compute the offset in the backend's implementation?

Reply:
It is not impossible, but I don't if it worth it.